free-3.md

```diff
@@ -57,12 +75,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think that a decent amount of the project would involve translating between the various social media bot API services and the GUI of this language. That could involve a direction translation with many similar functions or it could involve clever software design and a lot of pattern matching. Depending on the implementation, the time allocations for the project could vary. The similarities between different social media APIs would determine how much of the project would involve writing functions specific to one type of bot.
```

Comment:
One thing that I've also been wondering (not a rhetorical question, I really don't know): if you were to stick everything into a GUI would there really be a language design aspect? I feel like by sticking everything in an interface, the way the language is designed doesn't really matter to the user because it's hidden behind an interface. It'd be great if you could clarify this or just explain your thought process.